test/benchmark/fixtures.rb

```diff
@@ -27,6 +27,15 @@ class PostSerializer < ActiveModel::Serializer
   belongs_to :blog, serializer: BlogSerializer
   belongs_to :author, serializer: AuthorSerializer
 
+  link(:post_authors) { 'https://example.com/post_authors' }
+
+  meta do
+    {
+      rating: 5,
+      favorite_count: 10
+    }
+  end
```

Comment:
I wanted the transform benchmark to run on a serializer that has more than attributes to transform.